internal/util/rocksmq/server/rocksmq/rocksmq_impl.go

```diff
@@ -593,7 +593,7 @@ func (rmq *rocksmq) Consume(topicName string, groupName string, n int) ([]Consum
 
 	metaKey := constructCurrentID(topicName, groupName)
 	currentID, err := rmq.kv.Load(metaKey)
-	if err != nil {
+	if err != nil || currentID == "" {
 		log.Debug("RocksMQ: load " + metaKey + " failed")
 		return nil, err
 	}
```

Comment:
When `currentID` is empty string, the returned value is `nil, nil` which is not recommended.

Reply:
I'll fix this in next pr.

Reply:
btw, the log level in line 597 change to WARN may be better.

Reply:
I would suggest to return err if currentID not exist